docs/tutorial/3-advanced.md

```diff
@@ -404,10 +404,12 @@ a way to set any of that, we need to write our own heading show rule.
 >>>   ]
 >>> ]
 >>>
```

Comment:
would be nice if this or the newline below ... would be `>>>`. currently it renders like this:
```
...
= Introduction
...
```